### ISSUE TITLE
404 Page not found, on petition

### ISSUE THREAD

**Issue by CIwanow** [WhiteHouse/petitions#139]
Link to petition:  https://petitions.whitehouse.gov/petition/petition-donald-trump-calling-international-investigation-smolensk-plane-crash-april-10th-2010

Petition was working and got more than 40 000 signatures, now it's displaying error: "404 Page Not Found".

There's also error on developer's console: "Failed to load resource: the server responded with a status of 403 (Forbidden) petition-donald-trump-calling-international-investigation-smolensk-plane-crash-april-10th-2010"

**Comment by rbanffy** [comment]
I suppose the roadmap will only include solving issues like these after January 2021.

**Comment by rbanffy** [comment]
It seems it's back online

![image](https://user-images.githubusercontent.com/184714/39245114-b1a363b2-488a-11e8-9f42-e61664e5a1c4.png)

**Comment by CIwanow** [comment]
Yeah, great now it's online again (it was unavailable for few hours), but what's weird for me is that start date is  April 10, 2018 - originally it was April 13, 2018. So now end date is May 10, 2018 instead of May 13, so now we have 3 days less that previously. In case of any other problems - it's good place to raise issues? Or I should look for contact somewhere else? Here's screen with correct date:

![image](https://user-images.githubusercontent.com/11631496/39261362-84356920-48bc-11e8-9557-baa348edf7f7.png)

**Comment by rbanffy** [comment]
I'm not sure who's currently responsible for the website operations side. Tradition says the webmaster@ e-mail should route to someone responsible for operations, but I can't really speak on their behalf. If tradition doesn't work, the contact form (link on the page footer) would be our best option at the moment.

**Comment by CIwanow** [comment]
Ok, thanks for response. Someone fixed date :). BTW I thought that you work as developer in WH because you're member of "The White House organization"

**Comment by rbanffy** [comment]
My involvement is with the project, not WH operations.

**Comment by CIwanow** [comment]
ok, one more thing. Petition time is ended, and we got like 99% of needed votes. As I wrote petition was unavailable for 24h, so I think it's justified to add additional time ( we did not have a full month because of that). I have no idea if it's even possible, but I'm writing this because maybe someone from WH operations will reach this message (I contacted also from  official form on website).

https://petitions.whitehouse.gov/petition/petition-donald-trump-calling-international-investigation-smolensk-plane-crash-april-10th-2010

**Comment by pzabrocki** [comment]
Petition closed with 99008 signatures and really strange things took place during vote time. Apart from issues mentioned by @CIwanow (404 error, change of end date) I noticed several other problems that occured at least a few times:
1. The text of the petition was not visible
2. The form was not visible (people couldn't sign the petition!)
3. The progress bar showed 100%
4. Different people reporting different results at the same time (ok, let's say it's a local cache problem)
Strange, isn't it? :/
